src/IriReference/IriReference/IriReference.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+Copyright 2022 Equinor ASA
+
+This program is free software: you can redistribute it and/or modify it under the terms of version 3 of the GNU Lesser General Public License as published by the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
We need to have a licence file as well for this information, per organisation guidelines.